client/blocks/login/login-form.jsx

```diff
@@ -480,6 +483,10 @@ export class LoginForm extends Component {
 			return this.renderWooCommerce();
 		}
 
+		if ( isJetpackWooDnaFlow ) {
```

Comment:
The variables `isJetpackWooCommerceFlow` and `isJetpackWooDnaFlow` are really similar and it's hard to understand how they're different to someone new to this code. I'm trying to think how to make it clearer what's going on.

The biggest difference is that one flow involves a "Jetpack" branded flow that is different from the default Jetpack flow.

The "DNA" flow is using the Jetpack Connection package. This flow is essentially a "WP.com" connection flow.

Is DNA an official name that's being used? I don't see it in the JP connection package docs:

https://github.com/Automattic/jetpack/tree/master/packages/connection

What do you think? I couldn't really come up with anything better. I had some ideas like `isWooWPcomConnectionFlow` but that's not really clear either. Happy to leave it as is, especially if we add some comments in the config file, but wanted to mention in case you have a new idea as you're reading this.

Reply:
> Is DNA an official name that's being used?

No, it's not official, I made it up. I kind-of combined "Woo" with "Jetpack DNA" (which was the previous name of the Jetpack packaging efforts). The new flow didn't have a name, so i just used "Woo DNA" as a codename everywhere. Same way as "Gutenboarding" or "Frankenflow" (it's real, look it up), I expect that it will be just an internal name, and it being unique enough so it's easy to search for.

I'm not married to the "Woo DNA" name but I think it sounds cool :)

As to why it's `jetpackWooDnaFlow` instead of `wpcomWooDnaFlow` or something like that, Jetpack should still be part of the name. This file in particular, for example (`login-form.jsx`) is not specific to Jetpack, so being explicit that we're dealing with a Jetpack-related flow is important. Even if it's not Jetpack-branded, it's still a Jetpack connection flow internally, so the code shouldn't be trying to obfuscate that.